### PR TITLE
tweaked install UI

### PR DESCRIPTION
**Pull request by jasmingacic**
Signed-off-by: jasmingacic <jasmin.gacic@gmail.com>
image

### CODE REVIEW

**Review by donnalux — DISMISSED** [review]
Looks great!

File: cmd/kusk/cmd/deploy.go [hunk 1]
```diff
@@ -193,6 +193,7 @@ var deployCmd = &cobra.Command{
 				<-done
 			}
 		} else if e == nil {
+			fmt.Println(e)
```
Comment (kylehodgetts) [inline cmd/kusk/cmd/deploy.go:196]: Print `e` if it's nil?

**Review by kylehodgetts — CHANGES_REQUESTED** [review]
Will approve once all the commented out code is removed

File: cmd/kusk/cmd/deploy.go (hunk 1 above)
Comment (jasmingacic) [reply cmd/kusk/cmd/deploy.go:196]: duh :D

File: cmd/kusk/cmd/install.go [hunk 2]
```diff
@@ -118,62 +124,72 @@ var installCmd = &cobra.Command{
...
 				return err
 			}
-			fmt.Println("✅ API Server installed")
+			// kuskui.PrintInfo("⏳ waiting for API server to become ready...")
+			if err := utils.WaitForPodsReady(cmd.Context(), c, namespace, "kusk-gateway-api", time.Duration(5*time.Minute), "instance"); err != nil {
+				kuskui.PrintError("failed installing API Server")
+				reportError(err)
+				return err
+			}
+			// kuskui.PrintSuccess("API Server installed")
```
Comment (kylehodgetts) [inline cmd/kusk/cmd/install.go:168]: Needed?

File: cmd/kusk/cmd/install.go [hunk 3]
```diff
@@ -118,62 +124,72 @@ var installCmd = &cobra.Command{
...
 		if !noApi {
+			kuskui.PrintStart("installing API Server...")
 			if err := applyf(filepath.Join(dir, manifests_dir, "apis.yaml")); err != nil {
-				fmt.Println("❌ failed installing API Server", err)
+				kuskui.PrintError("failed installing API Server")
 				reportError(err)
 				return err
 			}
-			fmt.Println("✅ API Server installed")
+			// kuskui.PrintInfo("⏳ waiting for API server to become ready...")
```
Comment (kylehodgetts) [inline cmd/kusk/cmd/install.go:162]: Needed?

Comment (jasmingacic) [reply cmd/kusk/cmd/install.go]: no

File: cmd/kusk/cmd/install.go (hunk 2 above)
Comment (jasmingacic) [reply cmd/kusk/cmd/install.go]: nope

**Review by kylehodgetts — APPROVED** [review]

**Comment by jasmingacic** [comment]
@donnalux one more look before i merge this 
image

**Pull request merged**